matrix/splunk-otel-dotnet-metadata.yaml

```diff
@@ -0,0 +1,564 @@
+component: Splunk Distribution of OpenTelemetry .NET
+version: 1.0.2
+dependencies:
```

Comment:
I am not sure whether that's a good idea; this just simply duplicates build files. Are we supposed to list all libraries that we depend on? There's probably tens of them in case of the javaagent, and they're already there in the gradle files.

Reply:
I think that the goal is to define main dependencies here. I do not think that we need all of them.
The second part of the compatibility matrix is to generate it from source code, so you can parse/convert gradle file here.

Reply:
Main dependencies, meaning mostly upsteam OTel components? 

> The second part of the compatibility matrix is to generate it from source code, so you can parse/convert gradle file here.

Hmm, I suspect we probably could write some gradle code that generates that. I'm not a fan of that though.

Reply:
I was wondering if there's a better word we can use instead of dependencies. "components"? We want to highlight that these are feature enablers.